build-aux/m4/README.md

```diff
@@ -0,0 +1,8 @@
+## Autoconf archive
+
+All files in this folder are from the [Autoconf Archive](https://www.gnu.org/software/autoconf-archive/)
+([Github](https://github.com/autoconf-archive/autoconf-archive/tree/master/m4) may be ahead of the site).
+They are unmodified, but may not be the most recent serial.
```

Comment:
I don't think this  sentence adds much value, and is guaranteed to have to be modified in future.

Reply:
I (briefly) found it useful myself when I cloned the most recent archive, copied the files and did a git diff.